src/main/java/org/opentripplanner/apis/gtfs/datafetchers/fareImpl.java

```diff
@@ -15,8 +15,8 @@ public DataFetcher<Integer> cents() {
   }
 
   @Override
-  public DataFetcher<Iterable<FareComponent>> components() {
-    return environment -> (Iterable<FareComponent>) getSource(environment).get("details");
+  public DataFetcher<Iterable<Object>> components() {
```

Comment:
Does `components` refer to `FareComponent`s? If so, can/should the `components` method of `GraphQLFare` be removed too? Note that you removed a similar method from `ItineraryFares` in this PR, and I don't see a reference to that in the OTP-react-redux/OTP-UI code.

Reply:
We keep the API schema backwards compatible forever so I cannot remove it (even though it always returns an empty list). 

This is so that queries from old clients never fail even if they don't return any useful information.